### PR TITLE
Add content for vectors-matrix-ops/independence

### PR DESCRIPTION
**Pull request by razvand**
Add actual content to the skeleton of vectors-matrix-ops/independence.

### CODE REVIEW

File: chapters/vectors-matrix-ops/independence/reading/README.md [hunk 1]
```diff
@@ -1 +1,162 @@
...
+
+$$f(x,y) = f_X (x) f_Y (y) = \begin{cases} \displaystyle\frac{1}{4} &\text{if } 0 \leq x,y \leq 2\\ 0 & \text{otherwise} \end{cases}$$
+
+Also note that $f(x,y) \geq 0$ for all $(x,y) \in \mathbb{R}^2$ and
+
+$$\int\int f(x,y)dxdy = \int_{0}^{2}\int_{0}^{2} \frac {1}{4} dxdy = \displaystyle\frac {1}{4}.4 = 1$$
+
+It follows that
+
+$$E \[X Y\] &= \_-\^\_-\^ f(x,y) xy dxdy\ > &= \_y=0\^2\_x=0\^2 xy dxdy\ > &= \_y=0\^2 y (\_x=0\^2 x dx) dy\ > &= \_y=0\^2 y \_x=0\^2 dy\ > &= \_y=0\^2 y ( \^2 - \^2 ) dy\ > &= \_0\^2 y dy\ > &= \_0\^2 y dy\ > &= y\^2 \| \_0\^2\ > &= \^2\ > &= 1$$
```
Comment (teodutu) [inline chapters/vectors-matrix-ops/independence/reading/README.md]:
```suggestion
$$\begin{aligned} E [X Y] &= \_-\^\_-\^ f(x,y) xy dxdy\ > &= \_y=0\^2\_x=0\^2 xy dxdy\ > &= \_y=0\^2 y (\_x=0\^2 x dx) dy\ > &= \_y=0\^2 y \_x=0\^2 dy\ > &= \_y=0\^2 y ( \^2 - \^2 ) dy\ > &= \_0\^2 y dy\ > &= \_0\^2 y dy\ > &= y\^2 \| \_0\^2\ > &= \^2\ > &= 1 \end{aligned}$$
```
This doesn't render properly and I don't understand what it's trying to say so I can fix it. This is the only place in the repo where `\^` and `\_` are present.

**Review by teodutu — CHANGES_REQUESTED** [review]
I only left a partial suggestion. No idea how to completely fix the rendering issue caused by the line with the suggestion.

File: chapters/vectors-matrix-ops/independence/reading/README.md (hunk 1 above)
Comment (razvand) [reply chapters/vectors-matrix-ops/independence/reading/README.md]: I fixed it.

**Review by teodutu — APPROVED** [review]
Merry Christmas, @razvand!

**Pull request merged**